cvxpygen/cpg.py

```diff
@@ -62,8 +62,17 @@ def generate_code(problem, code_dir='CPG_code', solver=None, unroll=False, prefi
         shutil.copy(os.path.join(cvxpygen_directory, 'template', file), code_dir)
 
     # problem data
-    data, solving_chain, inverse_data = problem.get_problem_data(solver=solver, gp=False, enforce_dpp=True,
-                                                                 verbose=False)
+    solver_opts = {}
+    # TODO support quadratic objective for SCS.
+    if solver == 'SCS':
+        solver_opts['use_quad_obj'] = False
```

Comment:
Unrelated, but this makes me wonder if there's a better way to approach getting canonical data from CVXPY maybe methods on problems of the form:
 1. `get_qp_canon`
 2. `get_linear_obj_conic_constr_canon`
 3. `get_quad_obj_conic_constr_canon`
 
Since Clarabel, SCS, and MOSEK 10 all take 2 and 3 as inputs and 1 is good for all the QP solvers...